OsmAnd/src/net/osmand/plus/views/OsmandMapLayer.java

```diff
@@ -30,6 +30,23 @@ public void onRetainNonConfigurationInstance(Map<String, Object> map) {}
 	
 	public void populateObjectContextMenu(Object o, ContextMenuAdapter adapter) {}
 
+	/**
+	 * return true when this layer should handle single tap instead of context layer
+	 * @return
+	 */
+	public boolean isSingleTapResponder() {
```

Comment:
We can remove that method if we just follow onSingleTap single action click. onSingleTap returns true